src/agor/tools/dev_tooling.py

```
@@ -705,165 +377,130 @@ def create_development_snapshot(self, title: str, context: str) -> bool:
-        return escaped_content
+    Replaces triple backticks with double backticks to prevent formatting issues when embedding code within codeblocks.
+    """
+    return detick_content(content) # Use direct name
 
-    def detick_content(self, content: str) -> str:
-        """
-        Converts all triple backticks in the content to double backticks for safe embedding within a single codeblock.
 
-        Args:
-            content: The text in which to replace triple backticks.
+def detick_content(content: str) -> str:
```

Ruff (0.11.9)

Redefinition of unused `detick_content` from line 41

(F811)